From: Thomas Haller <thaller@redhat.com>
Date: Wed, 7 Oct 2020 08:39:56 +0200
Subject: platform/tests: adjust regeneration of test code for
 test_platform_ip_address_pretty_sort_cmp()

Since re-formatting our source code, the generated output no longer
matched the required formatting. Adjust it.

(cherry picked from commit 4fc79734985f661c7c28c56a558f00a795b9abf3)
(cherry picked from commit 07868a535f2454a7bdc7d1af9b96d8686d9f7dca)
---
 src/platform/tests/test-platform-general.c | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/src/platform/tests/test-platform-general.c b/src/platform/tests/test-platform-general.c
index f98f84c..aca4413 100644
--- a/src/platform/tests/test-platform-general.c
+++ b/src/platform/tests/test-platform-general.c
@@ -692,14 +692,16 @@ test_platform_ip_address_pretty_sort_cmp(gconstpointer test_data)
     }
 
     if (PRINT_RESULT) {
-        g_print("\n\n\t\t[%d] = (\n", TEST_DATA_I);
+        g_print("\n        [%d] = (", TEST_DATA_I);
         for (i = 0; i < ELM_SIZE * N_ADDRESSES;) {
-            g_print("\t\t\t\"");
+            if (i > 0)
+                g_print("\n               ");
+            g_print("\"");
             for (j = 0; j < 40 && i < ELM_SIZE * N_ADDRESSES; j++, i++)
                 g_print("%02x", addresses[i]);
-            g_print("\"\n");
+            g_print("\"");
         }
-        g_print("\t\t),\n\n");
+        g_print("),\n");
         return;
     }
 
